runtime/src/feature_set.rs

```diff
@@ -138,7 +143,7 @@ impl FeatureSet {
     /// All features enabled, useful for testing
     pub fn all_enabled() -> Self {
         Self {
-            active: FEATURE_NAMES.keys().cloned().collect(),
+            active: FEATURE_NAMES.keys().cloned().map(|key| (key, 0)).collect(),
```

Comment:
Any concerns about defaulting to zero here?

Reply:
Nope, I think that's correct.  Enabled since genesis